src/main/java/seedu/duke/exception/check/checkproperty/CheckPropertyMissingFlagException.java

```diff
@@ -0,0 +1,31 @@
+package seedu.duke.exception.check.checkproperty;
+
+//@@author ngdeqi
+import seedu.duke.exception.check.ParseCheckException;
+
+import static seedu.duke.CommandStructure.CHECK_PROPERTY_FLAGS;
+import static seedu.duke.Messages.MESSAGE_CHECK_PROPERTY_WRONG_FORMAT;
+import static seedu.duke.Messages.MESSAGE_MISSING_FLAG;
+
+/**
+ * Representation of an exception where the user omits the property flag for the check property command.
+ */
```

Comment:
You might want to rephrase this sentence to fits the standard indicated from the website.